internal/services/accountServiceImpl.go

```diff
@@ -0,0 +1,81 @@
+package user
+
+import (
+	"crud-golang/internal/dto"
+	"crud-golang/internal/models"
+	"crud-golang/internal/repositories"
+	"errors"
+	"golang.org/x/crypto/bcrypt"
+)
+
+type service struct {
+	repository repositories.Repository
+}
+
+func (s service) Create(user dto.AccountDto) (models.Account, error) {
+	hash, err := bcrypt.GenerateFromPassword([]byte(user.Password), bcrypt.DefaultCost)
+	if err != nil {
+		return models.Account{}, err
+	}
+
+	account := models.Account{
+		Password: string(hash),
+		Email:    user.Email,
+		Username: user.Username,
+	}
+	account, err = s.repository.Create(&account)
+	if err != nil {
+		return account, err
+	}
+	return account, err
+}
+
+func (s service) FindAll() (u []models.Account, err error) {
```

Comment:
Google has the best practice of passing contexts always. I would suggest you do the same. Especially when writing web servers, you would like to react to ctx.Done not to do db operation or any computation if it is not necessary